presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -277,3 +277,38 @@ The configuration properties of Presto C++ workers are described here, in alphab
 * **Default value:** ``0.25``
 
   See description for ``shared-arbitrator.memory-pool-min-free-capacity``
+
+Memory Checker Properties
+-------------------------
+
+The LinuxMemoryChecker extends from PeriodicMemoryChecker and is used for Linux systems only. 
+The LinuxMemoryChecker can be enabled by setting the CMake flag ``PRESTO_MEMORY_CHECKER_TYPE=LINUX_MEMORY_CHECKER``. 
+The following properties for PeriodicMemoryChecker are as follows:
+
+``systemMemPushbackEnabled``
```

Comment:
Are these the actual names of the config? The config values doesn't use came case.

They are called `system-mem-pushback-enabled` and so on. Please fix.

Reply:
Changing the names to match with https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/common/Configs.h#L275

Reply:
```
system-mem-pushback-enabled
system-mem-limit-gb
system-mem-shrink-gb
```